docs/decisions/0036-use-textarea-for-chat-content.md

```diff
@@ -81,4 +81,4 @@ Actually we used an `ExpandingTextArea` from `GemsFX` package so the content can
 as much space as it needs in the `ScrollPane`.
 
 About the selection and copying, this goes down to fundamental issue from JavaFX.
-`Text` and `Label` cannot be selected by any means.
+`Text` and `Label` as a whole or a part [cannot be selected and/or copied](https://bugs.openjdk.org/browse/JDK-8091644).
```

Comment:
Can you link ADR-0042 from here? (Similarly to what you did in ADR-0042 "More Information")

Reply:
Done!